classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -35,14 +35,12 @@ public function post( $request ) {
 			return $error;
 		}
 
-		if ( isset( $response->error ) ) {
-			$error = new WP_Error(
-				$response->error->code,
-				$response->error->message,
-				array( 'message' => $response->error->message )
+		if ( isset( $response->field_errors ) ) {
+			$this->logger->debug( 'Address validation errors: ' . implode( '; ', array_values( (array) $response->field_errors ) ), __CLASS__ );
```

Comment:
Note that this will need to be changed to `$this->logger->log` after https://github.com/Automattic/woocommerce-services/pull/1303 is merged.